swarm/pss/prox_test.go

```diff
@@ -139,11 +145,19 @@ func newTestData() *testData {
 	}
 }
 
-func (d *testData) init(msgCount int) {
+func (d *testData) init(msgCount int) error {
 	log.Debug("TestProxNetwork start")
 
 	for _, nodeId := range d.sim.NodeIDs() {
-		d.nodeAddrs[nodeId] = nodeIDToAddr(nodeId)
+		kadif, ok := d.sim.NodeItem(nodeId, simulation.BucketKeyKademlia)
```

Comment:
I would be happier to see here a higher level method to keep the core of the `for` simple.
Possibly that method should go on `simulation.Simulation` and be called something like `GetKademlia()`.